sass/components/_main-menu.scss

```diff
@@ -0,0 +1,166 @@
+@import "../abstracts/_mixins.scss";
+
+.menu-title {
+  text-align: center;
+  font-weight: 100;
+  margin-top: 5rem;
+  font-size: 3rem;
+  opacity: 0.3;
+  text-transform: uppercase;
+  padding-bottom: 2rem;
+}
+
+.project-title {
+  text-align: center;
+  font-family: Montserrat;
+  font-weight: 100;
+  margin-top: 100px;
+  font-size: 40px;
+  opacity: 0.3;
+  text-transform: uppercase;
+}
+
+.card-container {
+  height: 400px;
+  width: 300px;
+  border-radius: 2px;
+  position: relative;
+  overflow: hidden;
+  background: #fafafa;
+  cursor: pointer;
+  box-shadow: 0px 0px 25px rgba(0, 0, 0, 0.25);
+  transition: ease-in-out 0.3s;
+}
+
+.card-title {
+  position: absolute;
+  bottom: 65px;
+  width: 100%;
+  text-align: center;
+  color: #2f3640;
+  opacity: 0.9;
+  font-family: Montserrat;
+  font-size: 24px;
+  font-weight: normal;
+}
+
+.btn {
+  &--add-to-cart {
+    position: absolute;
+    opacity: 0;
+    width: 100%;
+    bottom: 30px;
+    text-transform: uppercase;
+    text-align: center;
+    padding-bottom: 5px;
+    color: #2f3640;
```

Comment:
Consider using variables for colors as this one is already re-used in 71st line. 😉